tests/testdata/generic_fault_injection.yaml

```diff
@@ -0,0 +1,31 @@
+targets:
```

Comment:
can't we put this one in code too? and enable it with some flag like `-useGenericFI`. with the helper I suggested above it wouldn't be that much code and it would be easy to edit.